.github/scripts/install-slices/install_slices.py

```diff
@@ -328,10 +382,25 @@ def main() -> None:
                 )
             else:
                 install_slice(
-                    full_slice_name(pkg.package, slice),
+                    pkg.package,
+                    slice,
                     cli_args.arch,
                     cli_args.release,
+                    has_copyright,
+                )
+
+        if not all(has_copyright.values()):
+            # Does the copyright file exist in the deb?
+            if deb_has_copyright_file(pkg.package):
+                err = "{} has a copyright file but it wasn't installed with: {}".format(
+                    pkg.package,
+                    ",".join(
+                        list(
```

Comment:
[Nit]: I think you don't need to create an in-memory list, `join` can consume the iterator directly.

Reply:
done 673670e